docs/features/automation.md

```diff
@@ -68,6 +68,17 @@ You can also create them manually by setting the following
 These rules are processed whenever you are importing recipes from websites or other apps
 and when using the simple ingredient input (shopping, recipe editor, ...).
 
+## Transpose Words
```

Comment:
Is there also documentation for never unit ? or is that the same as transpose words?

Reply:
they are both there...